src/homepage/howTo/custom-listeners.html

```diff
@@ -58,12 +58,16 @@ <h2><b>What is Mailbox Listener</b></h2>
                 </p>
                 <ul>
                     <li>
-                        <b>Group Registration</b>: If a Listener is registered as the Group Registration, it will receive all of events fired by James Mailbox System.
+                        <b>Group Registration</b>: a Group Registration will listen all events fired and delivery each event only once per group within a distributed topology.
```

Comment:
s/will listen all event/will listen **for** all event**s**
s/and delivery/and will deliver